manifests/machineconfigpool.crd.yaml

```diff
@@ -16,6 +16,10 @@ spec:
     description: When at least one of machine is not either not updated or is in the process of updating to the desired machine config.
     name: Updating
     type: string
+  - JSONPath: .status.conditions[?(@.type=="Degraded")].status
+    description: When the update for one of the machine is not progressig or the pool can't be configured due to an error.
```

Comment:
How about "When progress is blocked on updating one or more nodes, or the pool configuration is failing"?

Reply:
👍 thanks

Reply:
fixed